lib/testLauncher/SuitestLauncher.js

```diff
@@ -145,6 +148,7 @@ class SuitestLauncher {
 							[envVars.SUITEST_SESSION_TOKEN]: deviceAccessToken,
 							[envVars.SUITEST_DEVICE_ID]: deviceId,
 							[envVars.SUITEST_APP_CONFIG_ID]: appConfigId,
+							[envVars.SUITEST_DEBUG_MODE]: (inspectBrk || inspect || debugBrk) ? 'yes' : 'no',
```

Comment:
This condition repeats on line 156. Perhaps it makes sense to make a variable for this.